### DISCUSSION TITLE
Some Errors about dkl_mnist (Deep Gaussian Process)

### DISCUSSION THREAD

**Discussion by mohammadshams7201** [cornellius-gp/gpytorch#2050] [Q&A]
Dear Geoff Pleiss,
I hope you are staying healthy and safe.
When I ran dkl_mnist I faced the below error:
.
.
.
class DKLModel(gpytorch.GPModel):
.
.
.
------------------------------------------------
AttributeError: module 'gpytorch' has no attribute 'GPModel' 

How can I solve it?

**Comment by gpleiss** [comment]
What do you mean by "ran dkl_mnist"? Are you running the example notebook?

**Reply by mohammadshams7201** [reply]
Yes Sir. When I ran the notebook I received the problem!
![121](https://user-images.githubusercontent.com/102027547/176512065-cf61ec4e-4e3d-49d8-bf26-6989bde46d36.jpg)

**Reply by gpleiss** [reply]
That code does not come from one of our example notebooks. There is no `gpytorch.GPModel` object.

See https://docs.gpytorch.ai/en/stable/examples/06_PyTorch_NN_Integration_DKL/KISSGP_Deep_Kernel_Regression_CUDA.html or https://docs.gpytorch.ai/en/stable/examples/06_PyTorch_NN_Integration_DKL/Deep_Kernel_Learning_DenseNet_CIFAR_Tutorial.html for our DKL example notebooks.
